.pre-commit-config.yaml

```diff
@@ -0,0 +1,46 @@
+repos:
```

Comment:
Why there are two .pre-commit-config files?

Reply:
Good catch. Didn't mean to add one under the github dir